analysed-packages/libseccomp/version-2.5.3/libseccomp-2.5.3-SPDX2TV.spdx

```diff
@@ -0,0 +1,4905 @@
+SPDXVersion: SPDX-2.3
+DataLicense: CC0-1.0
+
+##-------------------------
+## Document Information
+##-------------------------
+
+DocumentNamespace: http://fossology-6674c79fbf-2qzn2/repo/SPDX2TV_libseccomp-2.5.3.tar.gz_1713879307.spdx
+DocumentName: /srv/fossology/repository/report
+SPDXID: SPDXRef-DOCUMENT
+
+##-------------------------
+## Creation Information
+##-------------------------
+
+Creator: Tool: fossology-4.4.0
+Creator: Person: Parian Golchin
+CreatorComment: <text>
+This document was created using license information and a generator from Fossology.
+It contains the license and copyright analysis of libseccomp 2.5.3.
+Please check "LicenseComments" for explanations of concluded licenses.
+</text>
+Created: 2024-04-23T13:35:08Z
+LicenseListVersion: 3.22
+
+##-------------------------
+## Package Information
+##-------------------------
+
+
+PackageName: libseccomp-2.5.3.tar.gz
+PackageFileName: libseccomp-2.5.3.tar.gz
+SPDXID: SPDXRef-upload159
+PackageDownloadLocation: NOASSERTION
+FilesAnalyzed: true
+PackageVerificationCode: f0d61452b04bef7c36159f9c6ab6288165892051
+PackageChecksum: SHA1: 039bb2242961ca11d1779039a18596f54ae5bdd8
+PackageChecksum: SHA256: 1e337ae9d8bab26641b17669a9871eaa10d4f3b474aaa4885d64b691a04614e3
+PackageChecksum: MD5: 4153c17013c0ede8bbf387b6115c235c
+PackageLicenseConcluded: NOASSERTION
+PackageLicenseDeclared: NOASSERTION
```

Comment:
Please add PackageLicenseDeclared: LGPL-2.1-only

Reply:
Done, thank you!